frontend/include/chpl/framework/resolution-error-classes-list.h

```diff
@@ -53,6 +53,7 @@ ERROR_CLASS(ModuleAsVariable,
             const uast::Module*)
 ERROR_CLASS(MultipleEnumElems, const uast::AstNode*, chpl::UniqueString, const types::EnumType*, std::vector<ID>)
 ERROR_CLASS(MultipleQuestionArgs, const uast::FnCall*, const uast::AstNode*, const uast::AstNode*)
+ERROR_CLASS(NestedClassFieldRef, const uast::AggregateDecl*, const uast::AggregateDecl*, const uast::AstNode*, ID)
```

Comment:
Can we make sure to format this like others that extend over 80 cols, if it does?